Core/Steps/SubSteps/ReleaseVersionAndMoveIssuesSubStep.cs

```diff
@@ -135,10 +132,20 @@ private bool ShouldMoveIssuesToNextVersion (string versionID, string nextVersion
       return false;
 
     _console.WriteLine("These are some of the issues that will be moved by releasing the version on jira:");
-    foreach (var issue in issuesToMove.Take(5))
-      _console.WriteLine($"'{issue.Key} - {issue.Fields.Summary}'");
+    PrintIssueListing(issuesToMove);
     _console.WriteLine("Do you want to move these issues to the new version and release the old one or just release the old version?");
 
     return _inputReader.ReadConfirmation();
   }
+
+  private void PrintIssueListing (IReadOnlyList<JiraToBeMovedIssue> issues, int displayAmount = 5, int maxLength = 75)
+  {
+    foreach (var issue in issues.Take(displayAmount))
+    {
+      var message = $"* '{issue.Key} - {issue.Fields.Summary}";
+      var length = message.Length > maxLength ? maxLength : message.Length;
+      _console.WriteLine(message.Substring(0, length) + "'");
```

Comment:
Since the key has a fixed width wouldn't it be easier to limit just the length of the summary? Right now it is hard to see what is actually printed since the parts are so split up. We are also missing the ellipsis `...` which makes the output a bit nicer.

```suggestion
     var summary = issue.Fields.Summary.Length <= maxLength
       ? issues.Fields.Summary
       : issues.Fields.Summary.Substring(0, maxLength - "...".Length) + "...";
      _console.WriteLine($"* '{issue.Key} - {summary}'");
```